lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -190,9 +190,9 @@ def get(self, request, *args, **kwargs):
         user_timezone_locale = user_timezone_locale_prefs(request)
         user_timezone = user_timezone_locale['user_timezone']
 
-        dates_tab_link = request.build_absolute_uri(reverse('dates', args=[course.id]))
-        if course_home_mfe_is_active(course.id):
-            dates_tab_link = get_learning_mfe_home_url(course_key=course.id, view_name='dates')
+        dates_tab_link = get_learning_mfe_home_url(course_key=course.id, view_name='dates')
+        if course_home_legacy_is_active(course.id):
+            dates_tab_link = request.build_absolute_uri(reverse('dates', args=[course.id]))
```

Comment:
I wonder if this should be if/else'd. We do it in some spots, but not here. I imagine it's a low cost operation so likely doesn't matter either way. I'm just musing aloud